notebooks/WFC3/point_spread_function/hst_point_spread_function.ipynb

```diff
@@ -0,0 +1,1551 @@
+{
```

Comment:
<p>Great work, just a minor suggestion:</p><p>I recommend use of context manager when working with FITS files.</p>

<pre class="ql-syntax" spellcheck="false">with fits.open(data_dir+hst_wfc3_f606w_file) as hdul:
    sci_data = hdul['SCI', 1].data
print('The image dimensions are:', np.shape(sci_data))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>